src/Language/PureScript/CoreFn/ToJSON.hs

```diff
@@ -0,0 +1,180 @@
+module Language.PureScript.CoreFn.ToJSON where
```

Comment:
Please add a short module comment here. Also, consider making the exports explicit (`moduleToJSON` and `annToJSON`, I think?)